agent/04_agent_configure.sh

```diff
@@ -511,11 +430,48 @@ if [[ "${NUM_MASTERS}" > "1" ]]; then
    set_api_and_ingress_vip
 fi
 
-if [[ $NETWORKING_MODE == "DHCP" ]]; then
-  generate_agent_config
-  generate_install_config
-else
-  generate_cluster_manifests
+MANIFESTS_PATH="${OCP_DIR}/cluster-manifests"
+
+mkdir -p ${MANIFESTS_PATH}
+if [ ! -z "${MIRROR_IMAGES}" ]; then
+  export MIRROR_PATH="${SCRIPTDIR}/${OCP_DIR}/mirror"
+  mkdir -p ${MIRROR_PATH}
+fi
+
+if [[ ${AGENT_USE_ZTP_MANIFESTS} == true ]]; then
+
+   generate_cluster_manifests
+
+ else
+
+  INSTALL_CONFIG_PATH="${OCP_DIR}"
```

Comment:
nit: would be nice to have a function like `generate_install_agent_config` for this block

Reply:
Good point, added